query/src/test/java/org/infinispan/query/blackbox/LocalCacheTest.java

```diff
@@ -298,23 +283,23 @@ public void testIteratorWithDefaultOptions() {
          found.next();
          assertFalse(found.hasNext());
       }
-      StaticTestingErrorHandler.assertAllGood(cache);
    }
 
    public void testExplain() {
       loadTestingData();
       CacheQuery<?> cacheQuery = createQuery("blurb:'Eats'", Person.class);
-      Explanation found = cacheQuery.explain(0);
+      Explanation found = cacheQuery.explain("S:BigGoat:0");
       assertNotNull(found);
-      StaticTestingErrorHandler.assertAllGood(cache);
    }
 
+   // TODO HSEARCH-3325 Restore support for full-text filters
```

Comment:
We are not going to support this anymore at API level: the filter will be part of the Ickle Query and if a user wants to deactivate it, just don't put in the query :)

Reply:
sounds great! I'm going to remove all the references to this feature.